src/lib/commandSetup.ts

```diff
@@ -33,13 +33,16 @@ export class Connections {
 
   get registry(): Registry {
     this.readyCheck();
-    if (!this._registry) throw new Error("No registry");
+    if (!this._registry)
+      throw new Error(
+        "No registry. This may be because you did not specify a private key with which to interact with the registry."
+      );
     return this._registry;
   }
 
   get validator(): Validator {
     this.readyCheck();
-    if (!this._validator) throw new Error("No registry");
+    if (!this._validator) throw new Error("No validator");
```

Comment:
Is there any kind of, "this might be because ...", explanation we could add here?

Reply:
Great point! I added a pointer.